streaming.md

```diff
@@ -138,47 +210,53 @@ FlinkPravegaWriter<MyClass> pravegaSink = FlinkPravegaWriter.<MyClass>builder()
+Every event written to a Pravega stream has an associated Routing Key.  The Routing Key is the basis for event ordering.  See the [Pravega documentation](http://pravega.io/docs/pravega-concepts/#events) for details.
 
-To establish the routing key for each event, provide an implementation of `io.pravega.connectors.flink.PravegaEventRouter` when constructing the writer.
+When constructing the FlinkPravegaWriter, to establish the Routing Key for each event, provide the below mentioned implementation:
+```
+io.pravega.connectors.flink.PravegaEventRouter` 
+```
 
 ### Event Time Ordering
-For programs that use Flink's event time semantics, the connector library supports writing events in event time order.   In combination with a routing key, this establishes a well-understood ordering for each key in the output stream.
 
-Use the `FlinkPravegaUtils::writeToPravegaInEventTimeOrder` method to write a given `DataStream` to a Pravega stream such that events are automatically ordered by event time (on a per-key basis). Refer [here](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/test/java/io/pravega/connectors/flink/FlinkPravegaWriterITCase.java#L93) for the sample code.
+For programs that use Flink's event time semantics, the connector library supports writing events in event time order. In combination with a Routing Key, this establishes a well-understood ordering for each key in the output stream.
+
+Use the following method to write a given `DataStream` to a Pravega stream such that events are automatically ordered by event time (on a per-key basis). ample code can be found [here](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/test/java/io/pravega/connectors/flink/FlinkPravegaWriterITCase.java#L93) for the sample code.
+
+```
+ FlinkPravegaUtils::writeToPravegaInEventTimeOrder
+ ```
 
 ### Writer Modes
 Writer modes relate to guarantees about the persistence of events emitted by the sink to a Pravega stream.  The writer supports three writer modes:
 1. **Best-effort** - Any write failures will be ignored hence there could be data loss.
-2. **At-least-once** - All events are persisted in Pravega.  Duplicate events
+2. **At-least-once** - All events are persisted in Pravega. Duplicate events
 are possible, due to retries or in case of failure and subsequent recovery.
 3. **Exactly-once** - All events are persisted in Pravega using a transactional approach integrated with the Flink checkpointing feature.
 
-By default, the _At-least-once_ option is enabled and use `withWriterMode(...)` option to override the value.
+By default, the At-least-once option is enabled and use `.withWriterMode(PravegaWriterMode.ATLEAST_ONCE)` option to override the value.

```

Comment:
is there any reason to remove `_At-least-once_` here? Also the parameter to withWriterMode(PravegaWriterMode.ATLEAST_ONCE) is not required since it could possible take 3 options.

Reply:
Accepted suggestion